test/models/list_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ListTest < ActiveSupport::TestCase
+  fixtures :lists
+
+  test "create list with empty title" do
+    list = List.new
+    assert list.invalid?
+    assert list.errors[:title].any?
+    assert !list.save
+    assert_equal "can't be blank", list.errors[:title].join('; ')
```

Comment:
@vlasiak, I would name the test according to what you expect to test: 
```ruby
test  'title presence'
```
Also, I believe that `assert list.errors[:title].any?` line is redundant.